indico/modules/events/contributions/controllers.py

```diff
@@ -95,8 +78,38 @@ class RHContributions(RHManageContributionsBase):
     """Display contributions management page"""
 
     def _process(self):
+        report_config = self.reporter.get_config()
+        if 'config' in request.args:
```

Comment:
I think the whole logic belongs to the reporter, something like `redirect_if_has_config` or something like that. Otherwise every report RH will have to implement the same logic.